sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/windowExpressions.scala

```diff
@@ -511,14 +511,15 @@ case class Lag(
 
 abstract class AggregateWindowFunction extends DeclarativeAggregate with WindowFunction {
   self: Product =>
-  override val frame: WindowFrame = SpecifiedWindowFrame(RowFrame, UnboundedPreceding, CurrentRow)
```

Comment:
@beliefer does `AggregateWindowFunction` support all kinds of window frames?

Reply:
Yes. `AggregateWindowFunction` support all frames.